astlib/astlib_intf.ml

```diff
@@ -1,10 +1,18 @@
 module type History = sig
+  (** Represents the history of AST grammars in the OCaml compiler. *)
   type t
 
+  (** Produces an alist of grammars, paired with their version numbers. *)
```

Comment:
```suggestion
  (** Produces an association list of grammars, paired with their version numbers. *)
```

Just to make things completely obvious. I'm not sure the term `alist` is widely used outside of Jane Street

Reply:
"alist" is a term common in lisp and schemes, but perhaps not outside that context. Anyway I'm happy with that change.